CHANGELOG.md

```diff
@@ -26,3 +26,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cybeark/conjur-openapi-spec#67](https://github.com/cyberark/conjur-openapi-spec/issues/67)
 - `all` query parameter supported on Roles endpoint.
   [cybeark/conjur-openapi-spec#68](https://github.com/cyberark/conjur-openapi-spec/issues/68)
+- Generic authenticator endpoints for authentication/other external authenticator requests.
```

Comment:
Let's split the "Generic authenticator endpoint that covers most Conjur platform authenticators" and "Endpoint to configure enabled Conjur authenticators via the API" (#66) into two separate lines in the change log.